asapdiscovery-docking/asapdiscovery/docking/scripts/run_docking_oe.py

```diff
@@ -552,7 +631,18 @@ def main():
         ]
         mp_args.extend(new_args)
 
+    if args.verbose:
+        if len(skipped) > 0:
+            logger.warning(f"Skipped {len(skipped)} receptor/ligand pairs")
+            for s in skipped:
+                logger.warning("Skipped pair: " + s)
+
+    logger.info(
+        f"MP args built, {len(mp_args)} total with {failures} failures, most likely due to skipped xtals"
+    )
```

Comment:
`len(mp_args) = 0` can happen in multiple situations and can be a result of different things (usually when no `xtals` are found). Might be good to throw an exception in this case, and suggest to the the user to turn on logging to debug.

Reply:
Have added